<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArduinoScope</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBoxSensors">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Sensor values</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="LCDsLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecord"/>
   <addaction name="actionScreenshot"/>
   <addaction name="separator"/>
   <addaction name="actionParameters"/>
   <addaction name="actionSavePath"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-record">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="toolTip">
    <string>Start recording</string>
   </property>
  </action>
  <action name="actionSavePath">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save path</string>
   </property>
   <property name="toolTip">
    <string>Choose the folder where the CSV files will be written</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
   <property name="toolTip">
    <string>Application parameters</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="toolTip">
    <string>Take a screenshot of the graph</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About this application</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
